Bindings/Python/tests/test_sockets_inputs_outputs.py

```diff
@@ -255,16 +255,18 @@ def print_model():
             reporter = osim.TableReporterVec3()
             reporter.setName('reporter')
             reporter.set_report_time_interval(0.1)
+
+            model.addComponent(reporter)
+
```

Comment:
@chrisdembia is it documented that reporters need to be added to the model before their inputs are wired? Is that necessary?

Reply:
It was necessary early on in the PR but I later removed that restriction (that's why there's `Input::_registeredChannels` now. I can try reverting this change.

Reply:
No need to, I mainly wanted to know if the documentation needed to match the new behavior.